INSTALL.md

```diff
@@ -44,6 +44,16 @@ sudo yum install -y \
     wget git
 ```
 
+On SLE/openSUSE
+```sh
+# Install RPM packages for dependencies
+sudo zypper install -y \
+  libseccomp-devel \
+  libuuid-devel \
+  openssl-devel \
+  cryptsetup sysuser-tools \
+  gcc go
```

Comment:
What versions of Go are available in the SLE/openSUSE repos? In general it appears distributions typically package versions of Go that are too old for the project, so we include instructions for installing from the official binaries

Reply:
In tumbleweed which a rolling release, it is 1.17. For leap and SLE 15.3 its 1.16 which should also be new enough.
For the older distro versions (SLE12.x and Leap 42.x) go1.17 is available through the devel project. So should I add also instructions how to add the repository of the devel project?

Reply:
1.16 is sufficient. It's pleasant to see a distribution with up to date Go packages :slightly_smiling_face: